workers-site/src/main.js

```diff
@@ -51,7 +51,19 @@ export async function handleRequest(event) {
       console.log('Handling redirect')
       return handleRedirect(request)
     }
-
+    // TODO remove this and just set the meta title/descriptions in Gatsby
+    if (pathname.includes('templates/pages')) {
+      //Grab the template's title from the registry
+      const templateId = pathname.replace(/.*pages\//, '')
+      const templateResp = await fetch(templateRegURL + templateId)
+      const templateJSON = await templateResp.json()
+      const metaInfo = {
+        title: templateJSON.title + ' - Cloudflare Workers Docs',
+        description: templateJSON.description,
+      }
+      // Rewrite all meta titles/descriptions the the correct title
+      return await new HTMLRewriter().on('meta', new MetaHandler(metaInfo)).transform(body)
```

Comment:
should we be rewriting the `title` tag too?

Reply:
Since we only get one HTMLRewriter that will be hard :(

Reply:
can we attach another handler on here?

```js
return await new HTMLRewriter()
  .on('meta', new MetaHandler(metaInfo))
  .on('title', new TitleHandler(metaInfo.title))
  .transform(body) 
```

Reply:
only the first transform handler will execute

Reply:
hey @victoriabernard92, i'm not sure i understand! you should be able to add multiple `.on` handlers, afaik. did this not work?

Reply:
Fixed :)